src/grid/index.js

```diff
@@ -135,13 +135,15 @@ const Grid = ( {
 							{ enableNewUi ? (
 								<div className="stc-grid-item-hover-button-wrapper">
 									<div className="stc-grid-item-hover-button-wrap">
-										<a
-											className="stc-grid-item-hover-button"
-											href={ item.link }
-											rel="noreferrer"
-										>
-											{ buttonLabel }
-										</a>
+										{ item?.link !== '#' ? (
+											<a
+												className="stc-grid-item-hover-button"
+												href={ item.link }
+												rel="noreferrer"
+											>
+												{ buttonLabel }
+											</a>
+										) : null }
```

Comment:
This directly removes the anchor tag when # is there. Is that expected?
Or do we need to just disable the link?

Reply:
Yes, It is expected.
We are removing the anchor tag if #.